src/MainWindow.vala

```diff
@@ -53,9 +56,22 @@ public class Mail.MainWindow : Gtk.ApplicationWindow {
         );
     }
 
+    static construct {
+        action_accelerators[ACTION_COMPOSE_MESSAGE] = "<Control>N";
+        action_accelerators[ACTION_REPLY] = "<Control>R";
+        action_accelerators[ACTION_REPLY_ALL] = "<Control><Shift>R";
+        action_accelerators[ACTION_FORWARD] = "<Ctrl><Shift>F";
+        action_accelerators[ACTION_MOVE_TO_TRASH] = "Delete";
+        action_accelerators[ACTION_MOVE_TO_TRASH] = "BackSpace";
+    }
+
     construct {
         add_action_entries (action_entries, this);
 
+        foreach (var action in action_accelerators.get_keys ()) {
+            ((Gtk.Application) GLib.Application.get_default ()).set_accels_for_action (ACTION_PREFIX + action, action_accelerators[action].to_array ());
```

Comment:
Over long line?  I would split in two using a temporary var for the application.